src/main/java/view/InputViewImpl.java

```diff
@@ -0,0 +1,39 @@
+package view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import message.RequestMessage;
+import validation.Validation;
+
+public class InputViewImpl implements InputView {
+
+  private final Validation validation = new Validation();
```

Comment:
싱글톤 적용하기. `getInstance()` static 메서드 만들어서 거기서 하나의 객체만 반환하도록 하기.
```suggestion
  private final Validation validation = Validation.getInstance();
```